FEXCore/Source/Interface/Core/OpcodeDispatcher/X87F64.cpp

```diff
@@ -107,6 +107,63 @@ void OpDispatchBuilder::FISTF64(OpcodeArgs, bool Truncate) {
   const auto Size = OpSizeFromSrc(Op);
 
   Ref data = _ReadStackValue(0);
+
+  // Need to check for overflow/NaN and set Invalid Operation flag
+  // The _Float_ToGPR_S/ZS operations don't set x87 exception flags in F64 mode
+  // So we need to do it manually here
+
+  // Extract the float value as GPR to check for special values
+  Ref DataGPR = _VExtractToGPR(OpSize::i64Bit, OpSize::i64Bit, data, 0);
```

Comment:
This is gonna be very slow, I *think* https://github.com/FEX-Emu/FEX/blame/fa0a54deb901cf1245357c2d834d9eefeb2083d7/FEXCore/Source/Interface/Core/OpcodeDispatcher/Vector.cpp#L2103 should match in behaviour though.

Reply:
This was removed from current version.